common/queue.go

```diff
@@ -109,7 +110,17 @@ type RedisQueue struct {
 func NewRedisQueue() Queue {
 	redisUrlString := os.Getenv("REDIS_PLUGIN_QUEUE_URL")
 	if redisUrlString == "" {
-		glog.Fatal("REDIS_PLUGIN_QUEUE_URL cannot be empty.\nexport REDIS_PLUGIN_QUEUE_URL=redis://host:port/db_number")
+
+		// try to connect via docker links if they exist.
```

Comment:
Hmm, not sure how I feel about having Docker related stuff in the code. It's easy enough to build up the environment variable when starting the containers, right?